Automattic/wp-calypso
# Marketplace: dynamic catalog cleanup

Related to https://github.com/Automattic/dotcom-forge/issues/1594

## Proposed Changes

* Removes `/marketplace/products-dynamic` call and relevant flag
* Enables `marketplace-fetch-all-dynamic-products` for all environments except production

## Testing Instructions



* Go to `plugins/browse/paid` and check that `Sensei Pro` is listed (not WC Courses one)
* Navigate to the plugin details page and make sure it loads.

## Pre-merge Checklist
